cmd/root.go

```diff
@@ -369,22 +371,22 @@ func getExecutableCommand(group, name string) (command.Command, error) {
 }
 
 // execute a cdt command
-func executeCommand(group, name string, args []string) error {
+func executeCommand(group, name string, args []string) (int, error) {
 	iCmd, err := getExecutableCommand(group, name)
 	if err != nil {
-		return err
+		return 404, err
```

Comment:
This doesn't look like standard Nix error code but rather http ones?

Reply:
Fixed in new commit